src/Symfony/Component/Console/Tests/Helper/ProgressBarTest.php

```diff
@@ -754,4 +754,21 @@ protected function generateOutput($expected)
 
         return "\x0D\x1B[2K".($count ? str_repeat("\x1B[1A\x1B[2K", $count) : '').$expected;
     }
+
+    public function testBarWidthWithMultilineFormat()
+    {
+        putenv('COLUMNS=10');
```

Comment:
this must be reset at the end of the test to avoid affecting other tests (including when the test fail, so use `try/finally`)